### PR TITLE
[env/github] remove MixPanel dependency

### PR DESCRIPTION
**Pull request by ermiaSparkle**
Remove Mixpanel

partially fixes https://github.com/sparkletown/internal-sparkle-issues/issues/839

**Before**
beforeMixpanel

**After**

afterMixpanel

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -7,7 +7,6 @@ import Bugsnag from "@bugsnag/js";
 import BugsnagPluginReact from "@bugsnag/plugin-react";
 import LogRocket from "logrocket";
 // eslint-disable-next-line no-restricted-imports
-import mixpanel from "mixpanel-browser";
```
Comment (0xdevalias) [inline src/index.tsx:10]: If you're moving the line it relates to, remove the associated comment as well.

Comment (ermiaSparkle) [reply src/index.tsx]: fix in 53838073e2551675ad8db79c5d249367d64c7700

**Review by mike-lvov — APPROVED** [review]
Looks good.

**Review by denisdimitrov — APPROVED** [review]
LGTM!

**Pull request merged**